static/src/js/components/SpatialDisplay/__tests__/SpatialDisplay.test.js

```diff
@@ -585,4 +585,28 @@ describe('SpatialDisplay component', () => {
       expect(result).toEqual('Longitude (190) must be between -180 and 180.')
     })
   })
+
+  describe('#validateBoundingBoxCoordinates', () => {
+    test('returns an error coordinates match each other.', () => {
+      const { enzymeWrapper } = setup()
+      const input = '-77.119,38.791'
+
+      const boundingBoxResult = enzymeWrapper.instance()
+        .validateBoundingBoxCoordinates([input, input])
+
+      expect(boundingBoxResult).toEqual('SW and NE points contain matching coordinates. Please use point selection instead.')
+    })
+  })
```

Comment:
```suggestion
  })
      test('returns an error coordinates match each other.', () => {
      const { enzymeWrapper } = setup()
      const input = '-91.119,38.791'

      const boundingBoxResult = enzymeWrapper.instance()
        .validateBoundingBoxCoordinates([input, input])

      expect(boundingBoxResult).toEqual('Latitude (-91.119) must be between -90 and 90.\nSW and NE points contain matching coordinates. Please use point selection instead.')
    })
```
I think we can just move that one into the same describe block wdyt?